.github/workflows/ci.yml

```diff
@@ -45,5 +45,8 @@ jobs:
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
 
+      - name: Publish SNAPSHOT
+        run: ./gradlew publishToMavenLocal -Pdisable_samples
```

Comment:
Remove that.

Reply:
It's needed, otherwise sample won't build. What do you suggest?

Reply:
Create a manual release on that branch and reference the tag?

Reply:
I think we need to merge plugin changes, make a build and then update the sample later. Not easy way around it.

Reply:
Done